README.md

```diff
@@ -50,20 +50,20 @@ as a subdirectory.
 > open build/Release/XiEditor.app
 ```
 
-Or `open XiEditor.xcodeproj` and hit the Run button.
+Or `open XiEditor.xcodeproj` and hit the `Run` button.
 
 ### Troubleshooting
 
 The most common cause of a failed build is an outdated version of `rustc`.
-If you've installed with rustup, make sure Rust is up to date by running
+If you've installed with [rustup](https://rustup.rs), make sure Rust is up to date by running
 `rustup update stable`.
 
 
 ## Configuration
 
 User settings are currently stored in files; the general preferences are
 located at `~/Library/Application Support/XiEditor/preferences.xiconfig`.
-This file can be opened from File > Preferences (⌘ + ,).
+This file can be opened from `File` > `Preferences (⌘ + ,)`.
```

Comment:
Mentioned in your other PR; you can use the markup `<kbd>⌘</kbd>` to have this rendered as a keycap: <kbd>⌘</kbd>.